c-example/README.md

```diff
@@ -0,0 +1,63 @@
+# C example
+
+This is the README file for spglib examples for C.
```

Comment:
```suggestion
This is an example of how to use Spglib through the C apis
```

Nitpick, but should we canonicalize `Spglib` or `spglib` or any other form for text names? Only the cmake project name we should stick to the common convention, otherwise, whichever style is more appealing